### DISCUSSION TITLE
Aria-busy causes label wrap to be messed up

### DISCUSSION THREAD

**Discussion by coezbek** [picocss/pico#622] [Help]
I am using Rails as a framework and it sets form elements to `aria-busy` when being submitted.

This has two unintended side-effects:

1. The whole form gets a busy indicator.
2. Label have nowarp enabled.

Example go to https://picocss.com/docs/forms - This is the way it looks by default:

![image](https://github.com/user-attachments/assets/21768642-530a-4dca-9e3c-9a5fb973f9bf)

Now use Chrome and add aria-busy="true" on the form. Now it looks like this:

![image](https://github.com/user-attachments/assets/26f370c3-80e4-4375-af92-7cd7c7a92100)

I have been able to work around this like this:

```
/* Prevent loading indicator from appearing on form elements */
form[aria-busy="true"]::before, turbo-frame[aria-busy="true"]::before {
  display: none;
}

form[aria-busy="true"] label {
  white-space: normal;
}
```

Is this something expected or a bug in Picocss?

**Comment by troelskn** [comment]
I ran in to the same issue. Seems to be caused by this line: https://github.com/picocss/pico/blob/6dc6489e69cc1f22daa938ebd38c02400032428c/scss/components/_loading.scss#L11

The commit that introduced it doesn't explain much. It just says "fix: loading component". I'm sure there is a reason for it, but I don't know what that might be. Any ideas @lucaslarroche ?

**Reply by Yohn** [reply]
There was a [pull request](https://github.com/picocss/pico/pull/625) open to fix this, and I've merged that update into [my fork of Pico CSS](https://github.com/Yohn/PicoCSS/)

The fix was to add the `form` to the line above the line you mentioned to prevent the loading from effecting the `form` tag.

```scss
#{$parent-selector} [aria-busy="true"]:not(input, select, textarea, html, form) {
  // ...
}
```
